javalib/src/main/scala/java/nio/channels/WriteOnlyFileChannel.scala

```diff
@@ -0,0 +1,19 @@
+package java.nio.channels
+
+//implements part of FileChannelImpl actually
+
+final class WriteOnlyFileChannel(val stream: Object, val handle: Long)
+    extends FileChannel /*Impl*/ {
+
+  private var append: Boolean = false;
+
+  def this(stream: Object, handle: Long, isAppend: Boolean) = {
+    this(stream, handle)
+    append = isAppend
+  }
+
+  def implCloseChannel(): Unit = ???
+  /*if (stream.isInstanceOf[Closeable]){
+            (stream.asInstanceOf[Closeable]).close()
+        }*/
```

Comment:
I removed the stubs for `FileChannel` altogether. I started from scratch in `java.nio` IIRC. I'll cherry-pick them back in `java.nio` if necessary.